joss_qmc/paper.md

```diff
@@ -0,0 +1,91 @@
+tags:
+  - Python
+  - SciPy
+  - statistics
+  - Quasi-Monte Carlo methods
+authors:
+  - name: Pamphile T. Roy
+    affiliation: 1
+    corresponding: true
+    orcid: 0000-0001-9816-1416
+  - name: Art B. Owen
+    affiliation: 2
+    orcid: 0000-0001-5860-3945
+  - name: Matt Haberland
+    affiliation: 3
+    orcid: 0000-0003-4806-3601
+affiliations:
+    - name: Quansight
+      index: 1
+    - name: Stanford University
+      index: 2
+    - name: California Polytechnic State University, San Luis Obispo, USA
+      index: 3
+date: 30 August 2022
+bibliography: paper.bib
+
+---
+
+# Summary
+
+
+# Statement of need
+
+NumPy random number generators (`numpy.random`) has become the standard way of
+sampling random numbers in the scientific Python ecosystem.
+These methods are fast and reliable, and the results are repeatable when a
+random seed is provided. However, sampling in high dimensions produces a lot of
+gaps and clusters of points. In integration problems, classical methods have a 
+low convergence rate meaning that a large sample size is required.
+
+By construction, Quasi-Monte Carlo (QMC) methods provide efficient, determinist
+(or not) and quality generators that can advantageously replace traditional
+methods. This can be decisive when the sampling size is limited or strong
+reproducibility guarantee is required. Also QMC methods are known to have
+better convergence rate than traditional Monte Carlo sampling (used by NumPy).
+
+QMC methods where added to SciPy [@virtanen2020scipy] after an extensive review
+and discussion period [@scipy2021qmc] that lead to a great collaboration
+between SciPy's maintainers and renown researchers in the field.
+
+The following set of features is currently available:
+
+- Sobol' and Halton sequences (scrambled and unscrambled),
+- Latin Hypercube Sampling (centred, optimized on $C^2$, orthogonal),
+- Poisson disk,
+- Multinomial and multivariate normal,
+- Discrepancy measures ($C^2$, wrap around, star-$L_2$, mixed),
+- Optimize a sample using $C^2$ or Lloyd-Max iterations,
+- Scaling utilities,
+- Fast numerical inverse methods to sample arbitrary distributions with QMC.
+
+Before the release of SciPy 1.7.0, the need for these functions was partially
+met in the scientific Python ecosystem by tutorials (e.g. blog posts)
+and niche packages, but the functions in SciPy have several advantages:
+
+- Prevalence: SciPy is one of the most downloaded scientific Python packages.
+  If a Python user finds the need for these statistical methods, chances are
+  that they already have SciPy, eliminating the need to find and install a
+  new package.
+- Speed: low level functions are written and optimized in Cython.
+- Easy-of-use: the function API reference and tutorials are thorough,
+  and the interfaces share common features and complement other SciPy functions.
+- Dependability: as with all SciPy code, these functions were rigorously
+  peer-reviewed, and unit tests are extensive.
```

Comment:
Hah when did I share my resampling paper? I don't remember sending it. (How does it look?)
Anyway, this is a little too similar/recognizable, as I plan to submit that one soon. You should probably write something like this bit without referring to this one.

Reply:
Sorry I was eyes dropping there 😅 Yes I have to rewrite this section. The whole paper as it is is mostly a patchwork so that we have a working base.